Cargo.toml

```diff
@@ -21,7 +21,7 @@ exclude = ["python"]
 resolver = "2"
 
 [workspace.package]
-version = "0.20.1"
```

Comment:
This is not relevant?

Reply:
this PR introduces a breaking change so bump the version